EmployeeLeaveTracking/EmployeeLeaveTracking.Data/DTOs/LeaveTypeWithTotalDaysDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace EmployeeLeaveTracking.Data.DTOs
+{
+    public class LeaveTypeWithTotalDaysDTO
+    {
+        public int LeaveTypeId { get; set; }
+        public string? LeaveTypeName { get; set; }
+        public int BookedDays { get; set; }
```

Comment:
Can BookedDays be floating in nature?

Reply:
Haven't added the Half Day Feature for leave request yet.